frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/common.embedding.unit.spec.tsx

```diff
@@ -100,37 +149,142 @@ describe("[OSS] embedding settings", () => {
       });
     });
   });
+
+  it("should link to quickstart for interactive embedding", async () => {
+    await setupEmbedding({
+      settingValues: {
+        "enable-embedding": false,
+        version: { tag: "v0.49.3" },
+      },
+    });
+    expect(getQuickStartLink()).toBeInTheDocument();
+    expect(getQuickStartLink()).toHaveProperty(
+      "href",
+      "https://www.metabase.com/docs/v0.49/embedding/interactive-embedding-quick-start-guide.html?utm_source=oss&utm_media=embed-settings",
+    );
+  });
+
+  it("should redirect users back to embedding settings page when visiting the full-app embedding page when embedding is not enabled", async () => {
+    await setup({
+      settings: [createMockSettingDefinition({ key: "enable-embedding" })],
+      settingValues: createMockSettings({ "enable-embedding": false }),
+      initialRoute: FULL_APP_EMBEDDING_URL,
+    });
+
+    expect(screen.getByText("Interactive embedding")).toBeInTheDocument();
+    expect(screen.getByText(/Embed dashboards, questions/)).toBeInTheDocument();
+  });
+
   describe("when the embedding is enabled", () => {
-    it("should allow going to static embedding settings page", async () => {
-      const { history } = await setupEmbedding({
-        settingValues: { "enable-embedding": true },
+    let history: Awaited<ReturnType<typeof setupEmbedding>>["history"];
```

Comment:
nit: should we extract `Awaited<ReturnType<typeof setupEmbedding>>["history"]` to its own type, as this is used 4 times and across files?

Reply:
Fixed in 77ca83b2bfda409cdacecd61b08f636f5fcef696